tool/server/routes.py

```diff
@@ -70,6 +68,9 @@
 CANCEL_URL = "/prescribe/cancel"
 RELEASE_URL = "/dispense/release"
 DISPENSE_URL = "/dispense/dispense"
+# TODO - think of a name for this
+DISPENSING_HISTORY_URL = "/dispense/history"
```

Comment:
lol think this is fine, except that it also gets the medicationRequest back. maybe `PRESCRIPTION_HISTORY`? lot of effort to change everywhere tho so think its fine